components/form/builder/src/Standard/index.js

```diff
@@ -92,8 +94,31 @@ const checkConstrainstsFactory = json => ({for: fieldID, all}) => {
     if (checkboxShouldBeTrueConstraint && !checkboxValue)
       errorMessages = [checkboxShouldBeTrueConstraint.message, ...errorMessages]
   }
-
   return errorMessages
 }
 
+const checkConstrainstsFactory = json => ({for: fieldID, all}) => {
+  let fieldsToValidate = {}
```

Comment:
`fieldsToValidate` is initialized as an object but later seems to be an array. Is this correct? Because the `.reduce` or `forEach` will be able to run only on arrays.

Reply:
thanks, first draft implementation was an object..